javascripts/AppConfig.js

```diff
@@ -0,0 +1,26 @@
+"use strict";
+
+app.run(function(FIREBASE_CONFIG){
+	firebase.initializeApp(FIREBASE_CONFIG);
+});
+
+app.config(function($routeProvider){
+	$routeProvider
+	.when("/contacts-favorites", {
+		templateUrl: "partials/contacts-favorites.html",
```

Comment:
@kstoops815 These should be `/contacts/favortites/` instead of `/contacts-favorites` (same for the next 2). And make sure your navbar links to them properly with the new names 😸